rest/route/subscription_test.go

```diff
@@ -244,3 +244,28 @@ func (s *SubscriptionRouteSuite) TestGetWithoutUser() {
 		_ = h.ParseAndValidate(ctx, nil)
 	})
 }
+
+func (s *SubscriptionRouteSuite) TestDisallowedSubscription() {
+	ctx := context.Background()
+	ctx = context.WithValue(ctx, evergreen.RequestUser, &user.DBUser{Id: "me"})
+	body := []map[string]interface{}{{
```

Comment:
I kind of like this because it more closely mimics what the client might send (since unmarshalling the struct itself is guaranteed to be in a valid format)